<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/Untitled.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>25</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Mode 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Mode 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Joystick</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Joystick</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Slider</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>DC Control: </string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>15</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>136</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Motor 1 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Motor 2 :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>RGB:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>0</y>
     <width>41</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
